web/gui/main.js

```diff
@@ -4897,8 +4897,8 @@ function handleSignInMessage(e) {
     cloudToken = e.data.token;
 
     netdataRegistryCallback(registryAgents);
-    if (e.data.redirectUrl) {
-        window.location.replace(e.data.redirectUrl);
+    if (e.data.redirectUri) {
```

Comment:
Could you name the js var `redirectURI`?
From cloud we do that.
Ex: `agentID`, `callbackURL`, `redirectURI`.

Reply:
absolutely! done